app/views/active_admin/devise/mailer/unlock_instructions.html.erb

```diff
@@ -1,7 +1,7 @@
 <p>Hello <%= @resource.email %>!</p>
 
-<p>Your account has been locked due to an excessive amount of unsuccessful sign in attempts.</p>
+<p>Your account has been locked due to an excessive number of unsuccessful sign in attempts.</p>
 
 <p>Click the link below to unlock your account:</p>
 
-<p><%= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @resource.unlock_token) %></p>
+<p><%= link_to 'Unlock my account', unlock_url(@resource, unlock_token: @token) %></p>
```

Comment:
Looks like Devise was using the former previously but got lost in the mix when we last updated this file. Both updates were back in 2013/2014. Based on this, the same change would be necessary for the reset password instructions email as well.

Reply:
Indeed! We essentially need to backport https://github.com/heartcombo/devise/commit/e947a9cbec77a06340aebb1517168e7e6cb2e6ad. Weird nobody noticed this before :open_mouth:.

Reply:
> Weird nobody noticed this before 😮.

I haven’t debugged why, but the weird thing was that the unlock link worked for me in `development`, but it didn’t in `production`. Maybe that contributed to the issue being unseen for a while. But I was surprised, too, and almost couldn’t believe it when I stumbled upon it^^

Still, happy to find some low hanging fruit(;

Reply:
I guess people have been fixing their templates, but not contributing the fix upstream? :man_shrugging:

Could you fix the other templates as well?

Reply:
I have added the changes for the reset password token, but haven't tested them. This is the second of the three files from the devise upstream change (https://github.com/heartcombo/devise/commit/e947a9cbec77a06340aebb1517168e7e6cb2e6ad). The third file (confirmation instructions) doesn't seem to be overwritten by ActiveAdmin, so there's no need to change it.

Reply:
@munen thanks for updating! If you have a moment can you please rebase? I'll approve and get this in next.

Reply:
I rebased this yesterday @javierjulio, it should be ready now!

Reply:
Thanks! I'll get this in now.